<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gridWindow</class>
 <widget class="QDialog" name="gridWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Array configuration dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>2D Vertical array configuration</string>
   </property>
   <widget class="QSpinBox" name="rowsSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of rows</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="columnsSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of columns</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height of blocks: </string>
    </property>
   </widget>
   <widget class="QLabel" name="z0Lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom elevation of starting block:</string>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="deltaHUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="deltaVUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total bottom elevation change in horizontal direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total bottom elevation change in vertical direction</string>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="lengthUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal distance between cell grids</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButtonV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical 2D array</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonH">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>26</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal 2D array</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxH">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>2D Horizontal array configuration</string>
   </property>
   <widget class="QSpinBox" name="rowsSpin_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of rows</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="columnsSpin_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of columns</string>
    </property>
   </widget>
   <widget class="QLabel" name="z0Lbl_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom elevation of starting block:</string>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="deltaHXUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="deltaHYUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total bottom elevation change in x direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total bottom elevation change in y direction</string>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="lengthXUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal distance between cell in x direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal distance between cell in y direction</string>
    </property>
   </widget>
   <widget class="UnitTextBox2" name="lengthYUnitBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnitTextBox2</class>
   <extends>QLineEdit</extends>
   <header>unittextbox2.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rowsSpin</tabstop>
  <tabstop>columnsSpin</tabstop>
  <tabstop>lengthUnitBox</tabstop>
  <tabstop>deltaHUnitBox</tabstop>
  <tabstop>deltaVUnitBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>gridWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>gridWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnsSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>columnsSpin_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnsSpin_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>columnsSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rowsSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rowsSpin_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rowsSpin_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rowsSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deltaHUnitBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>deltaHXUnitBox</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deltaHXUnitBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>deltaHUnitBox</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthUnitBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lengthXUnitBox</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthXUnitBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lengthUnitBox</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
